src/components/lime-ai-avatar/lime-ai-avatar.tsx

```
@@ -0,0 +1,61 @@
+            </Host>
+        );
+    }
+
+    private renderCircle(className: string) {
+        return (
+            <svg
+                class={className}
+                viewBox="0 0 100 100"
+                xmlns="http://www.w3.org/2000/svg"
+            >
+                <circle
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)